setup/etc/profile.d/gemdaqenv.sh

```diff
@@ -9,7 +9,7 @@ if [ "$XDAQ_ROOT" != "0" ]; then
     export XDAQ_ROOT=/opt/xdaq
     export XDAQ_DOCUMENT_ROOT=$XDAQ_ROOT/htdocs
     # only do this for regular users
-    # export XDAQ_DOCUMENT_ROOT=/data/xdaq/${USER}
```

Comment:
This change must be reverted, as it is only required for development and not suitable for releasing

Reply:
done

Reply:
This file was *not* listed in my instructions for cleaning up the old PR...